index.html

```diff
@@ -1287,13 +1263,27 @@
         reject <var>promise</var> with
         <code>"<a>NotSupportedError</a>"</code>, and abort these steps.
       </li>
-          <li>Let <var>adapter</var> be <code>null</code>.</li>
+      <li>
+        Let <var>adapter</var> be <code>null</code>.
+        Associate <var>adapter</var> with the current <code>Window</code>
```

Comment:
Right, I wanted to express that implementations associate the variable to the Window object, initially null. The reason was the suspended state of the adapter is affected by the calling script's Document's associated Window object's visibility and focus state. Perhaps I could even remove the sentence, without inducing uncertainty, since later steps will anyway need this association. So, we should not go into implementation details when not strictly needed.